src/siftsciencenet.sln

```diff
@@ -1,27 +1,33 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
-MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SiftScienceNet", "SiftScienceNet\SiftScienceNet.csproj", "{7D57EF7D-3AB6-458C-84C0-EB9867D58608}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{45BB0474-8EB4-43A0-8ECC-F5A38D46923E}"
-	ProjectSection(SolutionItems) = preProject
-		.nuget\packages.config = .nuget\packages.config
-	EndProjectSection
-EndProject
-Global
-	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU
-		Release|Any CPU = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{7D57EF7D-3AB6-458C-84C0-EB9867D58608}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{7D57EF7D-3AB6-458C-84C0-EB9867D58608}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{7D57EF7D-3AB6-458C-84C0-EB9867D58608}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{7D57EF7D-3AB6-458C-84C0-EB9867D58608}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SiftScienceNet", "SiftScienceNet\SiftScienceNet.csproj", "{7D57EF7D-3AB6-458C-84C0-EB9867D58608}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{45BB0474-8EB4-43A0-8ECC-F5A38D46923E}"
+	ProjectSection(SolutionItems) = preProject
+		.nuget\packages.config = .nuget\packages.config
+	EndProjectSection
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SiftScienceNetTests1", "SiftScienceNetTests1\SiftScienceNetTests1.csproj", "{B5AD0DA4-51C6-4D07-9C70-DC6900E362EA}"

```

Comment:
Just one thing, it fail to build because of a reference to the project SiftScienceNetTests1\SiftScienceNetTests1.csproj